recipes/ann/build.sh

```diff
@@ -0,0 +1,17 @@
+if [ "$(uname)" == "Darwin" ];
+then
+    export MACOSX_VERSION_MIN=10.9
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
```

Comment:
Do you need to downgrade the C++ version?

Reply:
No, I have removed the c++11 flag

Reply:
Can you try removing `-stdlib=libc++` and `-mmacosx-version-min` as well, I suspect they're also not needed and will be set correctly by cmake.

(If I'm wrong and `-mmacosx-version-min` is needed you should remove `${MACOSX_VERSION_MIN}` and use `${MACOSX_DEPLOYMENT_TARGET}`)

Reply:
Looks like they are not needed either, I removed them.